### PR TITLE
Event Blob is more common

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
I'm not sure this is a good idea but it's easier than fixing blob as a round type

### CODE REVIEW

File: code/modules/events/blob.dm [hunk 1]
```diff
@@ -1,11 +1,10 @@
 /datum/round_event_control/blob
 	name = "Blob"
 	typepath = /datum/round_event/blob
-	weight = 5
+	weight = 10
 	max_occurrences = 1
 
 	min_players = 20
-	earliest_start = 18000 //30 minutes
```
Comment (ChangelingRain) [inline code/modules/events/blob.dm]: You may still want to limit it to > 15 minutes

**Comment by coiax** [comment]
1) Aren't event blobs weaker?
2) Event blobs can't pick their starting location, they still use the blob landmarks.
3) If people can call the shuttle when a blob appears, then it's not going to be very interesting.

**Comment by ChangelingRain** [comment]
It wouldn't be hard to let them pick a location or make the event silent, I set up overminds to make that easy to do.

**Comment by LatD** [comment]
Late spawn blobs should be stronger than round start blobs, not weaker. People have better guns at that point anyway.

**Comment by Durkel** [comment]
Even more 30 minute rounds, I cry every time.

**Comment by Supermichael777** [comment]
blob is just not fun

**Pull request closed without merging**